### PR TITLE
update front page copy

### PR DESCRIPTION
**Pull request by adammichaelwood**
Addresses #117

#### What is included in this PR?

 - update "excerpt" (the text just below the HERO unit)
 - update the three feature call outs

#### Issues / Concerns / Possible Controversy

 - I made all the mentions of high profile users into links. I can see an argument against this, but I think it makes the social proof more powerful and interesting.
    - This caused a removal of Google as a mentioned user, as I couldn't find a really solid link.
    - Most of these are off-site links to resources we don't control. So link rot may become a problem in the future.

 - The third feature call out mentioned ODK 2, but in a way that suggested it was an update or new-hotness. I removed it. I think, currently, the entire home page talks meaningfully about things common to both suites.  

#### Other issues identified

Adding links on this page makes it clear that we need to programmatically make all outbound links open in new tabs (`target="_blank"`). This is easy to do with JS. (Added as #127)

### CODE REVIEW

**Comment by adammichaelwood** [comment]
Here are screen shots to make this easier to review if you can't get to a local build quickly:

screen shot 2018-05-25 at 1 09 33 pm
screen shot 2018-05-25 at 1 09 45 pm

File: index.html [hunk 1]
```diff
@@ -12,25 +12,25 @@
...
 feature_row:
   - image_path: /assets/images/design.png
     alt: "Build data collection forms using powerful tools."
-    title: "Build data collection forms using powerful tools."
-    excerpt: "Create forms to collect critical information using our easy-to-use, drag-and-drop design tools. From simple Q&A to powerful logic flows, we've got you covered."
+    title: "Build custom surveys and forms"
+    excerpt: "Use our drag-and-drop designer or our powerful Excel-based  tools to create forms as simple or as complex as you need them."
```
Comment (jbeorse) [inline index.html]: If we're opening it up to "as complex as you need them" it seems relevant to point out they can write their own code and modify the tools too, which has been a major feature of the ODK community. Maybe "Customize your surveys and workflows to be as simple or as complex as you need them: use our straightforward drag-and-drop designer, dig deeper with our powerful Excel-based tools, or program a completely customized solution."

File: index.html [hunk 2]
```diff
@@ -12,25 +12,25 @@
...
...ss-and-ifrc-visits-border-between-bolivia-and-brazil--72334/), [Kiva](https://www.kiva.org/blog/technology-in-the-field-the-future-of-microfinance), the [Carter Center](http://blog.cartercenter.org/2012/08/06/carter-center-on-leading-edge-of-technology-use-in-election-observation/), the [City of New York](https://opendatakit.org/2011/09/new-york-city-using-odk-for-health-operations/), [USAID](https://www.agrilinks.org/blog/partnership-open-source-technology-opens-door-improved-survey-data), [WHO](https://nafundi.com/blog/posts/improving-polio-campaign-quality-in-jordan-somalia-and-sudan/), [CDC](https://www.cdc.gov/ophss/chiic/projects/2014/ideas/using-open-data-kit-odk-for-rapid-data-collection-transport-visualization-and-management-of-malaria-data-in-siaya-county-kenya.html), the [Jane Goodall Institute](http://www.janegoodall.org/2016/11/21/monitoring-forests-2/), and [many others](https://forum.opendatakit.org/c/showcase). <br><br> From [election monitoring](http://getelmo.org/about/) to [tracking outbreaks](https://opendatakit.org/2015/01/collecting-50-million-data-points-across-200-projects-in-40-countries/),<br>Open Data Kit is the standard for in-the-field data collection."
```
[the remark this answers is not in the document]
Comment (adammichaelwood) [reply index.html]: I had found that first link already, but it seemed too much like a tutorial for this kind of showcase. I really like that second one, though, and will add it.

**Comment by adammichaelwood** [comment]
I took all of @jbeorse's suggestions, with a little tweaking. The place I deviated furthest was on the first feature call out, where I thought the suggestion was too long, compared to the other two items.

**Pull request closed without merging**